RoadMap_Insta/RoadMap_Insta/Cells/ActionPostTableViewCell/ActionPostTableViewCell.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ActionsViewController.swift
+//  RoadMap_Insta
+//
+//  Created by Dima Kovrigin on 25.10.2022.
+//
+
+import UIKit
+
+/// Ячейка с картинкой
+final class ActionPostTableViewCell: UITableViewCell {
+    
+    // MARK: - IBOutlet
+    
+    @IBOutlet private weak var avatarImageView: UIImageView!
+    @IBOutlet private weak var contentLabel: UILabel!
+    @IBOutlet private weak var postImageView: UIImageView!
+    
+    // MARK: - Lifecycle
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+    
+    // MARK: - Public methods
+    
+    func configure(model: ActionContent) {
+        contentLabel.attributedText = NSMutableAttributedString()
+            .bold("\(model.userName) ")
```

Comment:
```suggestion
            .bold("\(.userName) ")
```
Забываем слово model